### ISSUE TITLE
Digital stories - full suite of colours 

### ISSUE THREAD

**Issue by Norvard** [wellcometrust/wellcomecollection.org#926]
In order to use 4 unique brand colours across digital stories, each colour needs 3 shade variations in order to meet accessibility when applied to various elements within the build.

Here is a full cheat sheet that outlines what the colours are and where to use them.

![screen shot 2017-05-23 at 15 04 35](https://cloud.githubusercontent.com/assets/23660297/26358099/6815069e-3fc9-11e7-9104-cf4fa6e0fa0c.png)

**Comment by Norvard** [comment]
Updated after chatting to Captain @davidpmccormick 

![screen shot 2017-05-23 at 15 19 46](https://cloud.githubusercontent.com/assets/23660297/26358727/543c515c-3fcb-11e7-91a8-03a524ff6a5c.png)

**Comment by Norvard** [comment]
Updated. Slight mistake on Colour 02 Regular:

![screen shot 2017-05-24 at 10 34 23](https://cloud.githubusercontent.com/assets/23660297/26396972/ab6b2df8-406c-11e7-9c08-3671a9c8eec8.png)

**Comment by Norvard** [comment]
Just for clarity. The rules around choosing the right colour option with any given element inside a digital story (regular vs. darker):
- Regular is for all elements that do not require accessibility. So basically any non text element.
- Darker is for any text set on white or cream background colour
- When setting text on top of colour (regular), use our black (hex: 1d1d1d)
